include/mbedtls/config.h

```diff
@@ -2370,7 +2340,7 @@
  * \note When #MBEDTLS_CMAC_ALT is active, meaning that the underlying
  *       implementation of the CMAC algorithm is provided by an alternate
  *       implementation, that alternate implementation may opt to not support
- *       AES-192 or 3DES as underlying block ciphers for the CMAC operation.
+ *       AES-192 as underlying block ciphers for the CMAC operation.
```

Comment:
I think this line should not be changed: this is about CMAC, and we're only removing 3DES support in TLS, not in the rest of the library.

Reply:
Done.